code/game/turfs/simulated/river.dm

```diff
@@ -0,0 +1,78 @@
+#define RANDOM_UPPER_X 200
+#define RANDOM_UPPER_Y 200
+
+#define RANDOM_LOWER_X 50
+#define RANDOM_LOWER_Y 50
+
+/proc/spawn_rivers(target_z = 5, nodes = 4, turf_type = /turf/simulated/floor/plating/lava/smooth)
+	var/list/river_nodes = list()
+	var/num_spawned = 0
+	while(num_spawned < nodes)
+		var/turf/simulated/F = locate(rand(RANDOM_LOWER_X, RANDOM_UPPER_X), rand(RANDOM_LOWER_Y, RANDOM_UPPER_Y), target_z)
+
+		river_nodes += new /obj/effect/landmark/river_waypoint(F)
```

Comment:
Why are these landmarks instead of just turf list? Not seeing much use for these. Alternatively gc them.